src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const value = 40;
+  const minimumDiscount = 20;
+  const maximumDiscount = 50;
```

Comment:
Consider using more descriptive names for the constants. For example, 'DAILY_RATE', 'SHORT_TERM_DISCOUNT', 'LONG_TERM_DISCOUNT' would make the purpose of these values clearer.